server/src/org/apache/cloudstack/network/ssl/CertServiceImpl.java

```diff
@@ -14,7 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-package org.apache.cloudstack.network.lb;
+package org.apache.cloudstack.network.ssl;
```

Comment:
I got a little confuse here.
Did you create "org.apache.cloudstack.network.tls" or "org.apache.cloudstack.network.ssl"?